client/lbclient/boot/replication.js

```diff
@@ -23,17 +23,32 @@ module.exports = function(client) {
   };
 
   // setup model replication
+  var since = {push: -1, pull: -1};
```

Comment:
Nitpick: AFAIK, we use Node coding style where curly braces are surrounded by spaces: `{ push: -1, pull: -1 }`.